src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.hpp

```diff
@@ -273,7 +273,20 @@ class ReconstructionEngine_sequentialSfM : public ReconstructionEngine
 
   /// Export statistics in a JSON file
   void exportStatistics(double time_sfm);
-
+  
+  /**
+   * @brief Select the tracks whose are affected by the next triangulation. 
+   * @details A track is considered as triangulable if it is visible by at least one new reconsutructed 
+   * view and at least \c _minNbObservationsForTriangulation (new and previous) reconstructed view.
+   * @param[in] previousReconstructedViews The old reconstructed views.
+   * @param[in] newReconstructedViews The newly reconstructed views.
+   * @param[out] mapTracksToTriangulate A map with the tracks to triangulate and the observations to do it.
+   */
+  void getTracksToTriangulate(
```

Comment:
the method itself should be `const`, no?

Reply:
Absolutely

Reply:
Do you suggest me to use _inline_ on it?

Reply:
No I would not.
https://isocpp.org/wiki/faq/inline-functions#inline-and-perf